config_examples/config_freqai.example.json

```diff
@@ -75,7 +75,6 @@
             "weight_factor": 0.9,
             "principal_component_analysis": false,
             "use_SVM_to_remove_outliers": true,
-            "stratify_training_data": 0,
```

Comment:
is the removal of this intentional?

Reply:
Yes, setting it to 0 is equivalent to not setting it at all. I like the idea of keeping the example config as simple as possible.